virttest/utils_test/libvirt.py

```diff
@@ -1511,7 +1511,11 @@ def check_state():
         return utils_misc.wait_for(check_state, timeout)
 
 
-def check_result(result, expected_fails=[], skip_if=[], any_error=False):
+def check_result(result,
+                 expected_fails=[],
+                 skip_if=[],
+                 any_error=False,
+                 expected_match=[]):
```

Comment:
It is better to add all parameters explanation below

Reply:
As other parameters also missing explanation, could you please also add the left ones?